index.html

```diff
@@ -27,63 +31,71 @@
   <main>
 
     <section>
-      <img src="img/welcome.svg" alt="welcome to the gorgeous sedona" width="456" height="350">
+      <div class="welcome">
+        <img src="img/welcome.svg" alt="welcome to the gorgeous sedona" width="456" height="350" class="index-image">
+      </div>
       <h1>Седона — небольшой городок в Аризоне, заслуживающий большего! </h1>
-      <p>Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</p>
-      <ul>
+      <p class="five-seasons">Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</p>
+      <ul class="features-list">
         <li>
-          <h3>Настоящий городок</h3>
-          <p>— №1 —</p>
-          <p>Седона - не аттракцион для туристов, там течёт своя жизнь</p>
+          <section class="first-feature">
+            <h3>Настоящий городок</h3>
+            <p>— №1 —</p>
+            <p>Седона - не аттракцион для туристов, там течёт своя жизнь</p>
+          </section>
+
           <img src="img/image.jpg" alt="Жилье в Седоне" width="800" height="256">
-          <ul>
-            <li>
+
+          <ul class="feature-icons">
+            <li class="home-icon">
               <h3>Жилье</h3>
               <p>Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
             </li>
-            <li>
+            <li class="food-icon">
```

Comment:
во-первых li, это не icon, это item, во-вторых, лучше всегда отталкиваться от общего названия. Так удобней. И к тому же этот пункт имеет общие стили с другими пунктами, так что имеет смысл задать одинаковый класс, но так как у каждого пункта разные иконки, то нужно задать два класса:
```suggestion
            <li class="feature-item feature-item-food">
```